swagger-templates/python/api.mustache

```diff
@@ -149,14 +131,14 @@ class {{classname}}(object):
 
         query_params = {}
 {{#queryParams}}
-        if '{{paramName}}' in params:
+        if '{{paramName}}' in params and params['{{paramName}}'] is not None:
             query_params['{{baseName}}'] = params['{{paramName}}']
 {{/queryParams}}
 
         header_params = {}
 {{#headerParams}}
         if '{{paramName}}' in params:
-            header_params['{{baseName}}'] = params['{{paramName}}']
+            header_params['{{baseName}}'] = "Bearer %s" % (params['{{paramName}}'])
```

Comment:
NIT: `format` should be more typical than `%` nowadays, but they do the same thing. This would be `"Bearer {}".format(params['{{paramName}}'])`